pages/doc/integrations_tas_custom_app_metrics_howto.md

```diff
@@ -0,0 +1,75 @@
+---
+title: How to Monitor Custom App Metrics on TAS with Tanzu Observability
```

Comment:
Monitor Custom App Metrics on TAS

Reply:
Thanks, I see the pattern now: Even when the Docs Sidebar patter has "How  to ...", the Title does not.